scandium-core/src/main/java/org/eclipse/californium/scandium/config/DtlsConnectorConfig.java

```diff
@@ -113,10 +114,10 @@
 	/** store of the PSK */
 	private PskStore pskStore;
 
-	/** the private key for RPK and X509 mode */
+	/** the ECDSA private key for RPK and X509 mode */
```

Comment:
If the intention would be to limit the scope to ECDSA, I guess it would be expressed by using the EC specific subtypes. Right now I would just mention, that only EC is supported.